core/src/main/scala/za/co/absa/spline/harvester/plugin/registry/AutoDiscoveryPluginRegistry.scala

```diff
@@ -73,10 +73,22 @@ class AutoDiscoveryPluginRegistry(
     constr.newInstance(args: _*).asInstanceOf[Plugin]
   }
 
+  private def isPluginEnabled(pc: Class[Plugin]): Boolean = {
+    val pluginConf = conf.subset(pc.getName)
+    val isEnabled = pluginConf.getBoolean(EnabledConfProperty, EnabledByDefault)
+    if (!isEnabled) {
+      log.warn("Plugin {} is disabled in the configuration.", pc.getName)
```

Comment:
The message is evaluated lazily already no need for `{}` or is there?

Reply:
it's Loj4j. Don't you confuse it with Log4s? Log4j is Java API, there's no pass-by-name for method params.

Reply:
```
  protected def logWarning(msg: => String): Unit = {
    if (log.isWarnEnabled) log.warn(msg)
  }
```

Reply:
Ah, Ok, I can use another method `logWarn` that is Scala one with pass-by-name :)